build/parser.mjs

```diff
@@ -827,7 +827,10 @@ class Parser {
     )
       return;
 
-    item.isPrime = item.uniqueName.includes('Prime');
+    const isPrimeEquipment = item.name.endsWith('Prime') || item.name.startsWith('Prime');
+    const isPrimeMod = item.category === 'Mods' && item.name.startsWith('Primed');
+
+    item.isPrime = isPrimeEquipment || isPrimeMod;
```

Comment:
this also could be like this:
```suggestion
    item.isPrime = item.name.includes('Prime');
```

Reply:
nah, i purposely didn't do this, lol. there's items with prime, like `primed chamber` that are definitely _not_ prime.

there's a reason this wasn't how we did this. I promise i think through these things

Reply:
(ironically, i believe i also overrode primed chamber specifically so it would work). i think a category check should probably be used, but i can't remember if i had more context at some point.

Reply:
Oh right uhmm, there's a bunch of entries that should be added and some should be excluded

Reply:
primed chamber seems to be an odd mod, it can be an edge case but uhmmm, i will give some thought on how we could solve this

Reply:
Here it's the data i guatered

[isPrime.csv](https://github.com/WFCD/warframe-items/files/13862905/isPrime.csv)

Reply:
ooo, i didn't!

Reply:
yeah i just thought going the simple way and not having a bunch of weird conditional, but i think weird is part of warframe-items it makes part of it's beauty

Reply:
Could just use `toLowerCase`, would make it so that even if there's a casing change it would still all be lower case. 

Also as for primed mods I never saw them the same as Prime warframes/weapons. I always thought they were just mods that spent too much time in the void lore wise. A way for DE to upgrade needed mods without introducing more mods